### ISSUE TITLE
Process.binding error during `bunx --bun create-next-app`

### ISSUE THREAD

**Issue by Arctomachine** [oven-sh/bun#3351]
### What version of Bun is running?

0.6.9

### What platform is your computer?

Microsoft Windows NT 10.0.19044.0 x64, Linux 5.10.102.1-microsoft-standard-WSL2 x86_64 x86_64

### What steps can reproduce the bug?

Run `bunx --bun create-next-app@latest`

### What is the expected behavior?

Run (or start at least) next.js installer successfully

### What do you see instead?

```
TypeError: process.binding() is not supported in Bun. If that breaks something, please file an issue and include a reproducible code sample.
      at /tmp/create-next-app@latest--bunx/node_modules/create-next-app/dist/index.js:106:491
      at __nccwpck_require__ (/tmp/create-next-app@latest--bunx/node_modules/create-next-app/dist/index.js:325:43870)
      at /tmp/create-next-app@latest--bunx/node_modules/create-next-app/dist/index.js:299:3377
      at __nccwpck_require__ (/tmp/create-next-app@latest--bunx/node_modules/create-next-app/dist/index.js:325:43870)
      at /tmp/create-next-app@latest--bunx/node_modules/create-next-app/dist/index.js:299:2962
      at __nccwpck_require__ (/tmp/create-next-app@latest--bunx/node_modules/create-next-app/dist/index.js:325:43870)
      at /tmp/create-next-app@latest--bunx/node_modules/create-next-app/dist/index.js:325:15529
      at __nccwpck_require__ (/tmp/create-next-app@latest--bunx/node_modules/create-next-app/dist/index.js:325:43870)
      at /tmp/create-next-app@latest--bunx/node_modules/create-next-app/dist/index.js:325:44739
      at /tmp/create-next-app@latest--bunx/node_modules/create-next-app/dist/index.js:325:44781
      at (eval) (/tmp/create-next-app@latest--bunx/node_modules/create-next-app/dist/index.js:325:44785)
```

### Additional information

I looked into topmost file in error and here is the line. It happens for `const o=process.binding("fs");`
```js
*/var n=r(12294);function FragmentCache(e){this.caches=e||{}}FragmentCache.prototype={cache:function(e){return this.caches[e]||(this.caches[e]=new n)},set:function(e,t,r){var n=this.cache(e);n.set(t,r);return n},has:function(e,t){return typeof this.get(e,t)!=="undefined"},get:function(e,t){var r=this.cache(e);if(typeof t==="string"){return r.get(t)}return r}};t=e.exports=FragmentCache},90036:(e,t,r)=>{"use strict";const n=r(67097);const s=r(82361).EventEmitter;const i=r(57147);
const o=process.binding("fs");const a=o.writeBuffers;const c=o.FSReqWrap||o.FSReqCallback;
```

**Comment by Arctomachine** [comment]
Same issue with `create-payload-app` package. It also tries to binding('fs').

**Comment by birkskyum** [comment]
@Arctomachine , the `bunx --bun create-next-app@latest` appear to work now. Can you check again if it works for you too?

**Comment by Jarred-Sumner** [comment]
> @Arctomachine , the `bunx --bun create-next-app@latest` appear to work now. Can you check again if it works for you too?

I don't think it works, they depend on tty_wrap and fs in process.binding 
<img width="789" alt="image" src="https://github.com/oven-sh/bun/assets/709451/85cefaa5-dc5d-4f35-8419-f15468c1308e">

**Comment by Arctomachine** [comment]
Yes, at 0.8.1 issue still remains on my machine.


> @Arctomachine , the `bunx --bun create-next-app@latest` appear to work now. Can you check again if it works for you too?

How were you able to make it work on yours?

**Comment by birkskyum** [comment]
I see this:

```zig
- event compiled client and server successfully in 24 ms (18 modules)
2863 |         }
2864 |       }
2865 |     });
2866 |     var destroy2 = destroyImpl.destroy;
2867 | 
2868 |     Writable.prototype.destroy = function(err, cb) {
                                                                                     ^
TypeError: undefined is not an object
      at node:stream:2868:82
      at destroyer (node:stream:1465:44)
      at node:stream:3275:34
      at finishImpl (node:stream:3357:50)
      at finish (node:stream:3354:42)
      at node:stream:267:46
      at processTicksAndRejections (:1:2602)
2863 |         }
2864 |       }
2865 |     });
2866 |     var destroy2 = destroyImpl.destroy;
2867 | 
2868 |     Writable.prototype.destroy = function(err, cb) {
                                                                                     ^
TypeError: undefined is not an object
      at node:stream:2868:82
      at destroyer (node:stream:1465:44)
      at node:stream:3275:34
      at finishImpl (node:stream:3357:50)
      at finish (node:stream:3354:42)
      at node:stream:267:46
      at processTicksAndRejections (:1:2602)
```

**Comment by Electroid** [comment]
This is no longer an issue and has been fixed as of Bun v1.0.26.
